tools/sa_requester/lib/index.js

```diff
@@ -5,38 +5,64 @@ const fs = require('fs')
 const axios = require('axios')
 
 const CONTENT_TYPE_JSON = 'application/vnd.api+json'
+const FETCH_TIMEOUT = 5000
+const SERVICE_AGREEMENTS_PATH = '/v2/service_agreements'
+const ACCOUNT_BALANCE_PATH = '/v2/account_balance'
+
+function urlWithPath (t, path) {
+  const u = new URL(t)
```

Comment:
I receive the following here:

```
ReferenceError: URL is not defined
    at urlWithPath (~/go/src/github.com/smartcontractkit/chainlink/tools/sa_requester/lib/index.js:13:13)
    at Promise.all.oracleURLs.map.baseURL (~/go/src/github.com/smartcontractkit/chainlink/tools/sa_requester/lib/index.js:51:19)
    at Array.map (<anonymous>)
    at getOracleAddresses (~/go/src/github.com/smartcontractkit/chainlink/tools/sa_requester/lib/index.js:50:16)
    at SaRequester.run (~/go/src/github.com/smartcontractkit/chainlink/tools/sa_requester/lib/index.js:23:29)
    at SaRequester._run (~/go/src/github.com/smartcontractkit/chainlink/node_modules/@oclif/command/lib/command.js:29:31)
    at <anonymous>
```

Reply:
@thodges-gh my bad. I had my nodejs version set to 10.9. I've added the patches that will make it work in 8.x